futures/tests/async_await/pinned.rs

```diff
@@ -1,6 +1,7 @@
 use futures::stable::block_on_stable;
 use futures::executor::{block_on, ThreadPool};
 use futures::prelude::*;
+use futures::prelude::await;
```

Comment:
@ngg Is `await` not included automatically by the `*` import above?

Reply:
Without this the compiler gives an error "error[E0659]: `await` is ambiguous", "note: `await` could refer to the name imported here" (referring to the prelude), "note: `await` is also a builtin macro", "note: consider adding an explicit import of `await` to disambiguate". Explicitly importing it solves this.

Reply:
Ah, gotcha. Thanks for explaining!